composer.json

```diff
@@ -9,7 +9,7 @@
     }
   ],
   "require": {
-    "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+    "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

Comment:
Sorry, forgot to add, could you drop 8.0? We also do not support versions that do not receive any security updates anymore.

Reply:
Good idea!